packages/client/utils/getSearchQueryFromMeeting.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  TaskServiceEnum,
+  UpdatePokerScopeMutationResponse
+} from '../__generated__/UpdatePokerScopeMutation.graphql'
+
+const getSearchQueryFromMeeting = (
```

Comment:
The name may be misleading as we just want the query string and sometimes the filters

Reply:
I feel like it is ok, maybe consider inlining this into mutation if only needed for one particular case.